android-lint/src/main/java/timber/lint/TimberIssueRegistry.java

```diff
@@ -11,6 +11,11 @@
     return Arrays.asList(WrongTimberUsageDetector.getIssues());
   }
 
+  @Override
+  public int getMinApi() {
+    return 2;
```

Comment:
There's no constant for this?

Reply:
Doesn't seem like it. It's just a number.

Reply:
Yeah, but does not `LATEST` get bumped with each version? And if we don't need the latest API, we can default to a low min api.

Reply:
Then I guess we can just use `return ApiKt.CURRENT_API;`

Reply:
91cb640cab89d29e0d3072061676cc50399fd54f